specification/commerce/resource-manager/Microsoft.Commerce/preview/2015-06-01-preview/commerce.json

```diff
@@ -96,7 +96,7 @@
           "202": {
             "description": "Response indicating that a request has been accepted for processing. However, the processing has not been completed.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/definitions/ErrorObjectResponse"
             }
           },
```

Comment:
Java does not support 2 success response return 2 different model.

Reply:
This is what the service does and the behavior can't be changed.

Reply:
Sure, no problem. Java is not a required CI, and it won't block your PR.

Reply:
JS would fail to generate the SDK if 2 success response (200, 202) return 2 completely different model which means we can't release SDK either with this failure.

Reply:
Mary, I've added you to a team chat. Backend seems to mean this 202 as InProgress, not error.